File: a034(binary_convert)/Python/binary_convert.py

def to_binary(decimal_number: int) -> str:
    binary_number = ''
    while decimal_number != 0:
        binary_number += str(decimal_number % 2)
        decimal_number //= 2
    binary_number = ''.join(reversed(binary_number))
    return binary_number

while True:
    try:
        decimal_number = int(input())
        print(to_binary(decimal_number))
    except:
        break

File: a034(binary_convert)/C++/binary_convert.cpp
#include <iostream>
#include <algorithm>

std::string to_binary(int decimal_number) {
  std::string binary_number = "";
  while(decimal_number != 0) {
    binary_number += std::to_string(decimal_number % 2);
    decimal_number /= 2;
  }
  reverse(binary_number.begin(), binary_number.end());
  return binary_number;
}

int main(void) {
  int decimal_number;
  while(std::cin >> decimal_number) {
    std::cout << to_binary(decimal_number) << '\n';
  }
  return 0;
}
